integration/nwo/network.go

```diff
@@ -138,18 +138,19 @@ type Profile struct {
 
 // Network holds information about a fabric network.
 type Network struct {
-	RootDir                     string
-	StartPort                   uint16
-	Components                  *Components
-	DockerClient                *docker.Client
-	ExternalBuilders            []fabricconfig.ExternalBuilder
-	NetworkID                   string
-	EventuallyTimeout           time.Duration
-	SessionCreateInterval       time.Duration
-	MetricsProvider             string
-	StatsdEndpoint              string
-	ClientAuthRequired          bool
-	ChannelParticipationEnabled bool
+	RootDir                         string
+	StartPort                       uint16
+	Components                      *Components
+	DockerClient                    *docker.Client
+	ExternalBuilders                []fabricconfig.ExternalBuilder
+	NetworkID                       string
+	EventuallyTimeout               time.Duration
+	SessionCreateInterval           time.Duration
+	MetricsProvider                 string
+	StatsdEndpoint                  string
+	ClientAuthRequired              bool
+	ChannelParticipationEnabled     bool
+	ChannelParticipationFileRepoDir string
```

Comment:
Why are these on network? These are not network concepts but ordering feature flags. They should be moved to an appropriate position.

Reply:
These have been moved/removed as applicable. We'll no longer have the FileRepoDir configuration and instead will store the file repo relative to the Orderer.FileLedger.Location.